Main layout and demolink ready

Nyaroslav opened the pull request:
- [DEMO LINK](https://Nyaroslav.github.io/layout_html-form/)

irenhh commented on src/index.html:
```diff
@@ -1,15 +1,190 @@
+            <input
+              type="text"
+              id="name"
+              name="userName">
```
```suggestion
              name="userName"
            >
```

irenhh commented on src/index.html:
```diff
@@ -1,15 +1,190 @@
+            <input
+              type="checkbox"
+              id="agreement"
+              name="userAgreement">
```
some closing brackets are moved, some not...

rmnkk replied:
Got it. Next time will be more attentive.

Nyaroslav committed:
added layout

Nyaroslav committed:
fixed code structure

Nyaroslav committed:
demo link

etojeDenys reviewed (changes requested):
let`s try to cover all inputs into label instead of div

and add REPORT link

Nyaroslav committed:
div to label and [DEMO LINK](https://Nyaroslav.github.io/layout_html-form/)

Nyaroslav commented:
[TEST REPORT LINK](https://Nyaroslav.github.io/layout_html-form/report/html_report/)

Nyaroslav committed:
- [DEMO LINK](https://Nyaroslav.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Nyaroslav.github.io/layout_html-form/report/html_report/)
